classes/Language.php

```diff
@@ -279,8 +279,8 @@ public static function checkFilesWithIsoCode($iso_code)
     /**
      * @param string $iso_from
      * @param string $theme_from
-     * @param bool $iso_to
-     * @param bool $theme_to
+     * @param string|bool $iso_to
```

Comment:
```suggestion
     * @param string|false $iso_to
```
Same here (and probably in other places) Most of the time only `false` is used as default optional value That doesn't mean `true` can be used

Reply:
true & false  are not in primitive type for PHPDoc : https://docs.phpdoc.org/3.0/guide/guides/types.html